samples/dashboards/demo/accounting/demo.js

```diff
@@ -0,0 +1,637 @@
+                        rows: [{
+                            cells: [{
+                                id: 'rev-chart-kpi',
+                                width: '55%'
+                            }, {
+                                id: 'rev-forecast-kpi',
+                                width: '45%'
+                            }]
+                        }, {
+                            cells: [{
+                                id: 'cost-chart-kpi',
+                                width: '55%'
+                            }, {
+                                id: 'cost-forecast-kpi',
+                                width: '45%'
+                            }]
+                        }, {
+                            cells: [{
+                                id: 'res-chart-kpi',
+                                width: '55%'
+                            }, {
+                                id: 'res-forecast-kpi',
+                                width: '45%'
+                            }]
+                        }]
+                    },
+                    responsive: {
+                        large: {
+                            width: 470
+                        },
+                        medium: {
+                            width: '50%'
+                        },
+                        small: {
+                            width: '100%'
+                        }
+                    }
+                }, {
+                    id: 'stock-cell'
+                }]
+            }, {
+                cells: [{
+                    id: 'rev-chart',
+                    responsive: {
+                        large: {
+                            width: '50%'
+                        },
+                        medium: {
+                            width: '100%'
+                        },
+                        small: {
+                            width: '100%'
+                        }
+                    }
+                }, {
+                    id: 'cost-chart'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        cell: 'rev-chart-kpi',
+        type: 'KPI',
+        chartOptions: Highcharts.merge(commonGaugeOptions, {
+            title: {
+                text: 'Revenue (YTD)'
+            },
+            yAxis: {
+                max: 102,
+                tickPositions: [73, 83, 92, 102],
+                plotBands: [{
+                    from: 0,
+                    to: 73,
+                    className: 'null-band'
+                }, {
+                    from: 73,
+                    to: 83,
+                    className: 'warn-band'
+                }, {
+                    from: 83,
+                    to: 92,
+                    className: 'opt-band'
+                }, {
+                    from: 92,
+                    to: 102,
+                    className: 'high-band'
+                }]
+            },
+            series: [{}]
```

Comment:
Do we need it?

Reply:
This is because I'm updating the KPI series instead of adding it, but it's a good point since it won't be a lively updated dashboard anyway. I'll change it.